governance/polls/Increase Spark D3M Maximum Debt Ceiling to 1.2B - January 8, 2024.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Increase Spark D3M Maximum Debt Ceiling to 1.2B - January 08, 2024
+summary: Signal your support or opposition to increasing the Spark D3M Maximum Debt Ceiling to 1.2B
+discussion_link: https://forum.makerdao.com/t/spark-spell-proposed-changes/23298
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-01-08T16:00:00
+end_date: 2024-01-11T16:00:00
+---
+# Poll: Increase Spark D3M Maximum Debt Ceiling to 1.2B - January 8, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday,January 8 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to increasing the Spark [D3M](https://manual.makerdao.com/module-index/module-dai-direct-deposit) [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 400M from **800M** to **1.2B**.
```

Comment:
```suggestion
The community can vote in this poll to express support or opposition to making the following parameter change to the Spark Protocol [DAI Direct Deposit Module (D3M)](https://manual.makerdao.com/module-index/module-dai-direct-deposit) on SparkLend Ethereum:

- Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line)  by 400M DAI from 800M DAI to **1.2B DAI**.
```

Apply bold formatting only to the final number to clearly indicate the new value.